assets/js/country_support.js

```diff
@@ -69,14 +69,18 @@ function loadCountrySupportTable(elem, fetch) {
             isoCode,
             {
               name,
+              translated_names: translatedNames,
               country_code: countryCode,
               supports_sms: supportsSms,
               supports_voice: supportsVoice,
             },
           ]) => {
             const row = templateRow.cloneNode(true);
-
-            row.querySelector('[data-item=country]').innerText = `${name} (${isoCode})`;
+            let translatedName = name;
+            if (translatedNames) {
+              translatedName = translatedNames[locale] || name;
```

Comment:
Can we also update the spec in https://github.com/18F/identity-site/blob/main/spec/e2e/country_support_spec.js#L5 to have translated_names and test that those are used?

Reply:
I think the way we reformatted it should be good. all the logic is nwo in the IDP.